Permanent/Common/Base/SwiftUIViews/TextFieldViews/RoundStyledTextFieldView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  RoundStyledTextFieldView.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 19.12.2023.
+
+import SwiftUI
+
+struct RoundStyledTextFieldView: View {
+    @Binding var text: String
+    var placeholderText: String
+    var invalidField: Bool
+    var doneAction: (() -> Void)
```

Comment:
This one could create confusion as it's not called in IOS 14 and IOS 15. If I don't look into the class when I implement the code, this could create bugs. I don't see it used anywhere so I am not sure of the use of this.

Reply:
Added onCommit for ios14 textField doneAction closure.